### PR TITLE
[doc] Add a high-level summary of ownership transfer.

### PR DESCRIPTION
**Pull request by jadephilipoom**
Adds a step-by-step summary of how ownership transfer works and a few concrete examples to the ownership transfer documentation page. This is designed to complement the more detailed information already there and give newcomers some context.

### CODE REVIEW

File: doc/security/specs/ownership_transfer/_index.md [hunk 1]
```diff
@@ -12,18 +12,46 @@ ownership state of the device:
...
+These keys must include an RSA-3072 key pair for secure boot, an ECDSA-P256 key pair for unlocking ownership, and another ECDSA-P256 key pair for endorsing the next owner (see [Owner Keys]({{< relref "#owner-keys" >}})).
+    * The number of redundant keys stored is configurable up to around 2kiB of total key material.
+The new owner may choose, for example, to generate several RSA public/private key pairs.
+1. The new owner gives their public keys to the previous owner (which may be the manufacturer), who endorses them with a cryptographic signature.
+The Silicon Creator can also endorse new owners, although this will not happen often in practice.
+The endorser then creates an "ownership transfer payload", which includes the new owner’s public keys and their own ECDSA-P256 signature to authenticate the keys.
+1. The device should now be booted with a "transfer ownership" boot service request and with the ownership transfer payload to a special region of memory.
+During boot (specifically during the `ROM_EXT` stage), the device will check the signature on the payload against the endorser’s public key.
```
Comment (johannheyszl) [inline doc/security/specs/ownership_transfer/_index.md:30]: "against the endorser’s public key, which is either the previous owner or silicon creator."

**Review by moidx — APPROVED** [review]

File: doc/security/specs/ownership_transfer/_index.md (hunk 1 above)
Comment (jadephilipoom) [reply doc/security/specs/ownership_transfer/_index.md]: Added some points below that elaborate on this point, ptal?

**Review by johannheyszl — APPROVED** [review]
thx

File: doc/security/specs/ownership_transfer/_index.md [hunk 2]
```diff
@@ -12,18 +12,46 @@ ownership state of the device:
...
+**Scenario 1:** Immediately after manufacturing the device, the Silicon Creator assigns ownership to a known first Silicon Owner who has commissioned the device and prepared an ownership transfer payload in advance.
+
+**Scenario 2:** A Silicon Owner wants to sell their device to someone else.
+The buyer sends the seller an ownership transfer payload with their public keys.
+The seller signs the payload and sends the buyer a) their signature and b) the physical device in an unlocked state.
+The buyer can now complete the ownership transfer procedure to become the new owner.
+
+**Scenario 3:** A Silicon Owner wants to send their device back to the Silicon Creator for resale.
+They send it back in an unlocked state.
+The Silicon Creator can later endorse the ownership transfer payload of whoever buys the device, without further involvement from the previous owner.
```
[the remark this answers is not in the document]
Comment (jadephilipoom) [reply doc/security/specs/ownership_transfer/_index.md:56]: Yes -- I specify that it's sent back unlocked in the previous sentence, but I'll add some emphasis to that point.

[the remark this answers is not in the document]
Comment (johannheyszl) [reply doc/security/specs/ownership_transfer/_index.md:56]: sry, I see

[the remark this answers is not in the document]
Comment (jadephilipoom) [reply doc/security/specs/ownership_transfer/_index.md:57]: I think adding emphasis there was a good idea -- I added a new sentence now that explains how, although the Silicon Creator can endorse a new owner for an unlocked device, _only the current owner_ can unlock the device. I think it's good to make this point clear.

@moidx can you confirm that my understanding is correct here?

**Pull request merged**